app/controllers/cinema_halls_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CinemaHallsController < ApplicationController
+  def index
+    @cinema_halls = CinemaHall.all
+  end
+
+  def show
+    @cinema_hall = CinemaHall.find(params[:id])
```

Comment:
Consider not repeating yourself here - instead of writing `@cinema_hall = CinemaHall.find(params[:id])` in few lines, you could simply use:

```
before_action :set_cinema_hall, only: %i[ show edit update destroy ]
```

and then add a private method where you set and retrieve your cinema_hall:

```
def set_cinema_hall
  @cinema_hall = CinemaHall.find(params[:id])
end
```

This would allow you to completely remove line 7 (and others like it) from your code.

You can read about `before_action` [here](https://dannemanne.com/posts/ruby_on_rails_explained_before_action)

Reply:
That's really cool, I didn't think those simple CRUDs can get even simpler, thanks :)